packages/gatsby-theme-api-docs/src/components/api/resource/method/url-scopes-responses/scopes.js

```diff
@@ -1,33 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from '@emotion/styled';
 import { Markdown } from '@commercetools-docs/ui-kit';
 
-const Title = styled.div`
-  color: #999;
-`;
+import Title from './title';
 
 const Scopes = ({ scopes, title }) => {
   return (
     <div>
-      {title ? (
-        <Title>
-          <strong>{title}</strong>
-        </Title>
-      ) : null}
+      {title ? <Title>{title}</Title> : null}
 
-      <p>
-        {scopes.map((scope, index) =>
-          index === scopes.length - 1 ? (
-            <Markdown.InlineCode key={scope}>{scope}</Markdown.InlineCode>
-          ) : (
-            <span key={scope}>
-              <Markdown.InlineCode>{scope}</Markdown.InlineCode>
-              {', '}
-            </span>
-          )
-        )}
-      </p>
+      {scopes.map((scope, index) =>
+        index === scopes.length - 1 ? (
+          <Markdown.InlineCode key={scope}>{scope}</Markdown.InlineCode>
+        ) : (
+          <span key={scope}>
+            <Markdown.InlineCode>{scope}</Markdown.InlineCode>
+            {', '}
+          </span>
+        )
+      )}
```

Comment:
I think this can be simplified as:

```js
{scopes.map((scope, index) => (
  <>
    <Markdown.InlineCode>{scope}</Markdown.InlineCode>
    {index < scopes.length - 1 ? <span>{', '}</span> : null}
  </>
))}
```